notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb

```diff
@@ -782,12 +741,12 @@
       "source": [
         "## Deploy the model\n",
         "\n",
-        "Next, deploy your model for online prediction. To deploy the model, you invoke the `deploy` method, with the following parameters:\n",
+        "Next, deploy your model for online prediction. To deploy the model,  invoke the `deploy` method, with the following parameters:\n",
```

Comment:
remove extra space: To deploy the model,  invoke
to: To deploy the model, invoke